app/src/main/java/com/sdp13epfl2021/projmag/model/ImmutableProject.kt

```diff
@@ -84,7 +84,7 @@ data class ImmutableProject(
                             "students currently assigned but only $nbParticipant allowed to work for the project"
                 )
                 //!sectionsManager.isListValid(allowedSections) -> Failure("a section in the section list doesn't exist $allowedSections")
-                 //!tagsManager.isListOfStringsValidTags(tags) -> Failure("a tag in the tag list doesn't exist $tags")
+                //!tagsManager.isListOfStringsValidTags(tags) -> Failure("a tag in the tag list doesn't exist $tags")
```

Comment:
this comment can be removed?

Reply:
Because some projects don't have "valid" tags, kaou choose to disable theses tests until we modify all projects on firebase.